From 470b8dc75249e7c305c0bb03ba96515e42409332 Mon Sep 17 00:00:00 2001
From: Petr Nechaev <petr.nechaev@cogentembedded.com>
Date: Tue, 1 Oct 2019 16:04:15 -0400
Subject: [PATCH] ulcb-vb2.dtb: Remove extra nodes for sound through ak4613

Sound tested ok with default device tree (ulcb.dtsi), no need for
overrides in this file.
---
 arch/arm64/boot/dts/renesas/ulcb-vb2.dtsi | 44 -----------------------
 1 file changed, 44 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/ulcb-vb2.dtsi b/arch/arm64/boot/dts/renesas/ulcb-vb2.dtsi
index 50cdfd8993e7..89bc8c104201 100644
--- a/arch/arm64/boot/dts/renesas/ulcb-vb2.dtsi
+++ b/arch/arm64/boot/dts/renesas/ulcb-vb2.dtsi
@@ -38,13 +38,6 @@
 		};
 	};
 
-	snd_clk: snd_clk {
-		compatible = "fixed-clock";
-		#clock-cells = <0>;
-		clock-frequency = <24576000>;
-		clock-output-names = "scki";
-	};
-
 	vcc_sdhi3: regulator-vcc-sdhi3 {
 		compatible = "regulator-fixed";
 
@@ -92,28 +85,6 @@
 		enable-active-high;
 	};
 
-	/delete-node/sound;
-
-	rsnd_ak4613: sound@0 {
-		pinctrl-0 = <&sound_0_pins>;
-		pinctrl-names = "default";
-		compatible = "simple-audio-card";
-
-		simple-audio-card,format = "left_j";
-		simple-audio-card,name = "ak4613";
-
-		simple-audio-card,bitclock-master = <&sndcpu>;
-		simple-audio-card,frame-master = <&sndcpu>;
-
-		sndcpu: simple-audio-card,cpu@1 {
-			sound-dai = <&rcar_sound>;
-		};
-
-		sndcodec: simple-audio-card,codec@1 {
-			sound-dai = <&ak4613>;
-		};
-	};
-
 	lvds {
 		compatible = "panel-lvds";
 
@@ -205,13 +176,6 @@
 		function = "hscif4";
 	};
 
-	/delete-node/sound;
-
-	sound_0_pins: sound1 {
-		groups = "ssi01239_ctrl", "ssi0_data", "ssi1_data_a";
-		function = "ssi";
-	};
-
 	usb0_pins: usb0 {
 		groups = "usb0";
 		function = "usb0";
@@ -227,7 +191,6 @@
 		function = "usb30";
 	};
 
-
 	can0_pins: can0 {
 		groups = "can0_data_a";
 		function = "can0";
@@ -1559,13 +1522,6 @@
 	};
 };
 
-&rcar_sound {
-	pinctrl-0 = <&sound_clk_pins>;
-
-	/* Multi DAI */
-	#sound-dai-cells = <1>;
-};
-
 &ssi1 {
 	/delete-property/shared-pin;
 };
-- 
2.20.1

